blog.html

```diff
@@ -22,6 +22,94 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_380_release_announcement"></a>
+                    <a href="#apache_kafka_380_release_announcement">Apache Kafka 3.8.0 Release Announcement</a>
+                </h2>
+                xx July 2024 - Josep Prat (<a href="https://twitter.com/jlprat">@jlprat</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.8.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.8.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/documentation.html#upgrade_3_8_0">Upgrading to 3.8.0 from any version 0.8.x through 3.6.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>
+                    In the last release, 3.6,
+                    <a href="https://kafka.apache.org/38/documentation.html#tiered_storage">tiered storage</a> was released as early availability feature.
+                    In this release, Tiered Storage now supports clusters configured with multiple log directories (i.e. JBOD feature).
```

Comment:
Which KIP talks about this?

I know KRaft + JBOD was released as EA last release - 3.7. Is it GA today?

How does tiered storage relate?

Reply:
I'm repeating what's in the release notes for `3.8`. Maybe this is wrong. Could someone clarify? https://github.com/apache/kafka/blob/trunk/docs/upgrade.html#L31

Reply:
We should probably mention both separately and explicitly.

Further, we should probably think of a way to track in-flight GAs. We have a few features that were in GA in parallel - tiered storage, the new consumer group protocol, JBOD in KRaft. It's hard to track the status of these things

Reply:
@stanislavkozlovski Are we sure JBOD in KRaft is not GA?

Reply:
I'm asking because it has been a bit of a mystery how features are determined production ready in Apache Kafka.

Reply:
My understanding, especially wrt JBOD in KRaft, is that we may flag new features as EA if we release them with serious bugs or limitations that users may find surprising.

I think the concept of "production ready", or GA, arises naturally from the negation of the term EA. I don't know if these terms should mean anything more than just "not EA".

Reply:
pushed a fix

Reply:
@soarez - that's my understanding too